Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -514,6 +521,34 @@ static void activate(GApplication* application, WPEToolingBackends::ViewBackend*
 #endif
 }
 
+static gboolean parseWindowSize()
+{
+    if (!windowSize)
+        return FALSE;
+
+    gchar** windowSizes = g_strsplit(windowSize, "x", 2);
+    if (!windowSizes || !windowSizes[0] || !windowSizes[1]) {
+        g_warning("Failed to parse --size command line argument - use \"<size>x<height>\" without any additional whitespace.");
```

Comment:
With the option callback you have a `GError` to set instead of `g_warning`

Reply:
Done.